### ISSUE TITLE
Softmax functionality fail with mobile config

### ISSUE THREAD

**Issue by wok1909** [PSAL-POSTECH/PyTorchSim#136]
Mobile config
- num vector lane: 8
- spad size: 32K per vector lane

Softmax 
- input size: [16, 512, 512]
- dim: -1

![Image](https://github.com/user-attachments/assets/361018eb-48e5-400d-9dff-176eecc34f61)
